packages/bitcore-node/test/integration/websocket.integration.ts

```diff
@@ -104,4 +112,59 @@ describe('Websockets', function() {
     expect(hasSeenTxEvent).to.equal(true);
     expect(hasSeenCoinEvent).to.equal(true);
   });
+
+  it('should get wallet events', async () => {
+    const authKey = new PrivateKey();
+    const pubKey = authKey.publicKey.toString('hex');
+    const authClient = new Client({ baseUrl: 'http://localhost:3000/api', authKey });
+
+    const payload = { method: 'socket', url: 'http://localhost:3000/api' };
+    const authPayload = { pubKey, message: authClient.getMessage(payload), signature: authClient.sign(payload) };
+    const chain = 'BTC';
+    const network = 'regtest';
+    const roomPrefix = `/${chain}/${network}/`;
+    const address = '2MuYKLUaKCenkEpwPkWUwYpBoDBNA2dgY3t';
+    socket.emit('room', roomPrefix + 'wallet', authPayload);
```

Comment:
can you add a test for `/wallets` with more that one hosted wallet?

Reply:
Yeah, will do that after I finish this ripple pr with transforms